dist/images/start-ipsec.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -euo pipefail
+
+podNames=`kubectl get pod -n kube-system -l app=ovs -o 'jsonpath={.items[*].metadata.name}'`
+for pod in $podNames; do
+  caPod=$pod
+  break
+done
+
+echo " initing CA $caPod "
+kubectl exec -it $caPod -n kube-system -- ovs-pki init --force > /dev/null
+
+for pod in $podNames; do
+  echo " initing privkey,req,cert file on pod $pod "
+  systemId=$(kubectl exec -it ${pod} -n kube-system -- ovs-vsctl get Open_vSwitch . external_ids:system-id | tr -d '"' | tr -d '\r')
+
+  kubectl exec -it $pod -n kube-system -- ovs-pki req -u $systemId --force > /dev/null
```

Comment:
Why output to `/dev/null` instead of stdout

Reply:
Because the logs look messy, so I didn't let it output to stdout,